<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StaticIPForm</class>
 <widget class="QWidget" name="StaticIPForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="applyButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>270</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QLabel" name="staticIP">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>120</y>
     <width>481</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Static IP: 172.0.0.1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Gateway">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>170</y>
     <width>481</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Gateway: 172.0.0.1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="IPMask">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>110</y>
     <width>311</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="activatedLabel">
   <property name="geometry">
    <rect>
     <x>164</x>
     <y>90</y>
     <width>471</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Active</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Name">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>481</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Static IP settings</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="GatewayMask">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>160</y>
     <width>311</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Subnet">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>220</y>
     <width>481</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Subnet mask: 172.0.0.1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="SubnetMask">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>210</y>
     <width>311</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>10</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="moveLeftButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>70</width>
     <height>380</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="moveRightButton">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>90</y>
     <width>70</width>
     <height>380</height>
    </rect>
   </property>
   <property name="text">
    <string>-&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
